config/locales/en/support_user_mailer.yml

```
@@ -11,7 +11,7 @@ en:
 
         If you have a DfE Sign-in account, you can use it to sign in:
 
-        [sign_in_url](%{sign_in_url})
+        %{sign_in_url}
```

Why have you removed the link? It should be the following, no?

```suggestion
        [%{sign_in_url}](%{sign_in_url})
```